github-email-notifications/githubconsumer/githubconsumer.py

```diff
@@ -143,7 +159,19 @@ def fmt_pr(self, comment):
             project=self.project, **comment)
         msgid, threadid = self._msg_id(
             comment['repo'], comment['msgid'], comment['pr_num'])
-        self._send_email(subject, body, msgid, threadid)
+
+        attachments = ()
+        if comment['pr_action'] in {'opened', 'synchronize'}:
+            # send PR as patch in attachment
+            patch_data = self._get_patch("{pr_url}.patch".format(**comment))
```

Comment:
I wonder if we should be permissive here and send the e-mail (maybe with a warning) in case we can't fetch the patches, instead of aborting the operation